src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseFVM.cpp

```diff
@@ -175,6 +175,110 @@ void CompositionalMultiphaseFVM::assembleFluxTerms( real64 const dt,
+                                            phaseOutfluxAccessor.toNestedView(),
+                                            compOutfluxAccessor.toNestedView() );
+  } );
+
+  // Step 3: finalize the (cell-based) computation of the CFL numbers
+  real64 localMaxPhaseCFLNumber = 0.0;
+  real64 localMaxCompCFLNumber = 0.0;
+  forTargetSubRegions( mesh, [&]( localIndex const targetIndex, ElementSubRegionBase & subRegion )
+  {
+
+    arrayView2d< real64 const > const & phaseOutflux = subRegion.getReference< array2d< real64 > >( viewKeyStruct::phaseOutfluxString() );
+    arrayView2d< real64 const > const & compOutflux = subRegion.getReference< array2d< real64 > >( viewKeyStruct::compOutfluxString() );
+
+    arrayView1d< real64 const > const & volume = subRegion.getElementVolume();
+    arrayView1d< real64 const > const & porosityRef =
+      subRegion.getReference< array1d< real64 > >( viewKeyStruct::referencePorosityString() );
+
+    arrayView2d< real64 const > const & compDens = subRegion.getReference< array2d< real64 > >( viewKeyStruct::globalCompDensityString() );
+
+    ConstitutiveBase const & solid = getConstitutiveModel( subRegion, solidModelNames()[targetIndex] );
+    arrayView2d< real64 const > const & pvMult =
+      solid.getReference< array2d< real64 > >( ConstitutiveBase::viewKeyStruct::poreVolumeMultiplierString() );
+
+    MultiFluidBase const & fluid = getConstitutiveModel< MultiFluidBase >( subRegion, m_fluidModelNames[targetIndex] );
+    arrayView3d< real64 const > const & phaseVisc = fluid.phaseViscosity();
+
+    RelativePermeabilityBase const & relperm = getConstitutiveModel< RelativePermeabilityBase >( subRegion, m_relPermModelNames[targetIndex] );
+    arrayView3d< real64 const > const & phaseRelPerm = relperm.phaseRelPerm();
+    arrayView4d< real64 const > const & dPhaseRelPerm_dPhaseVolFrac = relperm.dPhaseRelPerm_dPhaseVolFraction();
+
+    KernelLaunchSelector2< CFLKernel >( m_numComponents, m_numPhases,
+                                        subRegion.size(),
+                                        volume,
+                                        porosityRef,
+                                        pvMult,
+                                        compDens,
+                                        phaseRelPerm,
+                                        dPhaseRelPerm_dPhaseVolFrac,
+                                        phaseVisc,
+                                        phaseOutflux,
+                                        compOutflux,
+                                        localMaxPhaseCFLNumber,
+                                        localMaxCompCFLNumber );
+  } );
+
+  real64 const globalMaxPhaseCFLNumber = MpiWrapper::max( localMaxPhaseCFLNumber );
+  real64 const globalMaxCompCFLNumber = MpiWrapper::max( localMaxCompCFLNumber );
+
+  GEOSX_LOG_RANK_0( "Max phase CFL number: " << globalMaxPhaseCFLNumber );
+  GEOSX_LOG_RANK_0( "Max component CFL number: " << globalMaxCompCFLNumber );
```

Comment:
Not sure if we want these logged at verbosity level 0 (which is normally just time stepping progress)... level 1 usually includes all the details of a time step. Although one could argue that this information is relevant to analyze time stepping behavior (cuts, etc.). Just something to consider.

Also, since we can have multiple physics solvers in a simulation, maybe consider prefixing this output with solver name:
```suggestion
  GEOSX_LOG_RANK_0( getName() << ": Max phase CFL number: " << globalMaxPhaseCFLNumber );
  GEOSX_LOG_RANK_0( getName() << ": Max component CFL number: " << globalMaxCompCFLNumber );
```

Reply:
I agree, I added the `getName()` and I changed the log level to 1 for the CFL numbers.